src/diffusers/models/unet_2d_condition.py

```diff
@@ -733,6 +734,38 @@ def _set_gradient_checkpointing(self, module, value=False):
         if hasattr(module, "gradient_checkpointing"):
             module.gradient_checkpointing = value
 
+    def enable_freeu(self, s1, s2, b1, b2):
+        r"""Enables the FreeU mechanism as in https://arxiv.org/abs/2309.11497.
```

Comment:
If you can also propagate these changes to the other docstrings that'd be awesome!